src/Graphics/src/Graphics/Platforms/iOS/UIImageExtensions.cs

```diff
@@ -27,17 +28,36 @@ public static UIImage ScaleImage(this UIImage target, float maxWidth, float maxH
 
 		public static UIImage ScaleImage(this UIImage target, CGSize size, bool disposeOriginal = false)
 		{
-			UIGraphics.BeginImageContext(size);
-			target.Draw(new CGRect(CGPoint.Empty, size));
-			var image = UIGraphics.GetImageFromCurrentImageContext();
-			UIGraphics.EndImageContext();
-
-			if (disposeOriginal)
+			if (!OperatingSystem.IsIOSVersionAtLeast(17))
```

Comment:
Require this validation on more places (essentials etc):

`D:\a\_work\1\s\src\Essentials\src\Screenshot\Screenshot.ios.cs(89,4): error CA1416: This call site is reachable on: 'MacCatalyst' 13.1 and later. 'UIGraphics.EndImageContext()' is unsupported on: 'maccatalyst' 17.0 and later. `